kustomize/commands/build/reorderoutput.go

```diff
@@ -33,7 +34,11 @@ func validateFlagReorderOutput() error {
 	}
 }
 
-func getFlagReorderOutput() krusty.ReorderOption {
+func getFlagReorderOutput(flags *flag.FlagSet) krusty.ReorderOption {
+	isReorderSet := flags.Changed(flagReorderOutputName)
+	if !isReorderSet {
+		theFlags.reorderOutput = string(krusty.ReorderOptionUnspecified)
```

Comment:
I think we need to return here, otherwise won't the default value be matched in the case statement below?

Reply:
No strong opinion! I'll make it return immediately.